config.mk

```diff
@@ -1,2 +1,2 @@
-SOURCES= core/sjcl.js core/aes.js core/bitArray.js core/codecString.js core/codecHex.js core/codecArrayBuffer.js core/codecBase64.js core/sha256.js core/ccm.js core/ccmArrayBuffer.js core/ocb2.js core/gcm.js core/hmac.js core/pbkdf2.js core/random.js core/convenience.js 
```

Comment:
This was just generated by `make distclean`, must've been removed previously without rebuilding?